File: CHAPTER 4 PRACTICE SET/problem_1.c
/*Write a program to print multiplication table of a given number n.*/

#include <stdio.h>
int main()
{

    int i = 1;
    int n;
    printf("enter value of n \n");
    scanf("%d", &n);

    for (i = 1; i <= 10; i++)
    {
        printf(" %d x %d = %d \n", n, i, n * i);
    }

    return 0;
}

File: PYTHON/CHAPTER 6/problem 1.py
# Write a program to find the greatest of four numbers entered by the user.

a = int(input("enter 1st number : "))
b = int(input("enter 2nd   number : "))
c = int(input("enter 3rd number : "))
d = int(input("enter 4th number : "))

if (a >b and a > c and a > d ) :

    print("1st is greatest number ")

elif (b > a and b > c and b > d) :

    print("second is greatest number ")

elif (c > b and c> a and c > d) :

    print("third is greatest number ")

elif (d >  b and d > c and d > a) :

    print("fourth is greatest number ")

else :
    print("error")







